client/src/components/span-table/spanHeader.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { useSelector } from 'react-redux';
+import { useAppDispatch } from '../../lib/hooks';
+import '../../styles/spanTable.scss'
+import { selectSourceMap } from '../sourceMapSlice';
+import { selectSpanMap } from './spanMapSlice';
+import { changeRenderView, RenderType } from './spanMapSlice';
+import { useAppSelector } from '../../lib/hooks';
+import { selectSearchTraceResult } from '../searchBarSlice';
+
+/** 
+   * Defines the contents of the Trace Table Headers
+   * @Remarks Manages spacing between elements via a CSS Grid defined in traceTable.scss
+   */
+
+
+ 
```

Comment:
Lots of whitespace here. Comment also seems inaccurate.

Reply:
removed